priv/templates/ct_suite.erl

```diff
@@ -0,0 +1,33 @@
+-module({{name}}_SUITE).
+
+-include_lib("common_test/include/ct.hrl").
+-include_lib("stdlib/include/assert.hrl").
+
+-compile(export_all).
```

Comment:
Quite controversial discussion may start with this one.

Reply:
If you decide to keep `export_all` is it better to have `-compile([export_all, nowarn_export_all]).` instead of different directives?